ui/lib/core/addon/components/form-field.hbs

```diff
@@ -28,10 +36,21 @@
               @onChange={{this.setAndBroadcast}}
               @groupValue={{(get @model this.valuePath)}}
               @disabled={{and @attr.options.editDisabled (not @model.isNew)}}
+              data-test-radio="{{if (or (eq @attr.type 'boolean') val.value) val.value val}}"
             />
-            <label for={{or val.value val}} class="has-left-margin-xs">
-              {{or val.value val}}
-            </label>
+            <div class="has-left-margin-xs">
+              <label
+                for="{{or val.value val}}"
+                class="has-left-margin-xs {{if this.hasRadioSubText 'has-text-black is-size-7'}}"
```

Comment:
I don't think the classes here should change depending on whether or not there is subtext. If only some options have subtext then the different label text classes will look strange beside one another. 
```suggestion
                class="has-left-margin-xs has-text-black is-size-7"
```

Reply:
@hellobontempo I agree, but I do think it's important that these radio group's look as closely to the designs and HDS as possible I updated the conditional to check every `possibleValue` to see if the radio has both label and subtext so we know that it's a radio group.